### ISSUE TITLE
Failed to adjust affinity of [executable] (Access Denied)

### ISSUE THREAD

**Issue by TROPACLONE** [rcmaehl/NotCPUCores#123]
This error occurs while trying to use NotCPUCores for Squad (Steam game). Don't know why or if it happens for other programs. I think it's not anti-cheat related, since I closed the game's anti-cheat and the issue persisted. I also executed NotCPUCores as admin and my Windows account is on admin mode. 
OS: Windows 10

**Comment by rcmaehl** [comment]
I'll look into this. Wouldn't surprise me if it's AntiCheat releated as I believe another anticheat like EAC had issues with this in the past.

**Comment by TROPACLONE** [comment]
I highly doubt it since I also tried using in on Notepad.exe and got the same issue, got that "Failed to etc" in multiple instances and instantly exited. I don't really know why it happens or even if it's a NCC issue or a problem with my OS.

**Comment by rcmaehl** [comment]
Is [executable] the executable for Squad or is it things such as svchost?

**Comment by TROPACLONE** [comment]
it's things like svchost. 
https://www.youtube.com/watch?v=WYoSCgQf_0M
it's basically what happens to that guy but I get an instantaneous exit, that "Pause/Break to Stop" doesn't even appear.

**Comment by rcmaehl** [comment]
> it's things like svchost. https://www.youtube.com/watch?v=WYoSCgQf_0M it's basically what happens to that guy but I get an instantaneous exit, that "Pause/Break to Stop" doesn't even appear.

Things like svchost is normal but the instantaneous exit is not. To clarify, when you mean "Exit" the entire program closes out or no?

**Comment by TROPACLONE** [comment]
no, I just get the " Exit. Restoring Previous State... / Done ", that's what I meant by "Exit".
To sum things up, I "Assign" Squad, get all those "Failed to adjust affinity of [executable] (Access Denied)" followed by "Exit. Restoring Previous State... / Done".
